modules/tensor_mechanics/examples/coal_mining/cosserat_elastic.i

```diff
@@ -0,0 +1,569 @@
+# Strata deformation and fracturing around a coal mine
+#
+# A 2D geometry is used that simulates a transverse section of
+# the coal mine.  The model is actually 3D, but the "x"
+# dimension is only 10m long, meshed with 1 element, and
```

Comment:
I just spent a while trying to formulate this in 2D.  The Cosserat stuff just isn't allowing me to do it.  I just set disp_x as an AuxVariable (and always zero) so i haven't got those disp_x DOFs hanging around at least.